core/src/main/java/hivemall/utils/hadoop/HiveUtils.java

```diff
@@ -231,6 +231,16 @@ public static boolean isListOI(@Nonnull final ObjectInspector oi) {
         return category == Category.LIST;
     }
 
+    public static boolean isNumberListOI(@Nonnull final ObjectInspector oi) {
+        return isListOI(oi)
+                && isNumberOI(((ListObjectInspector) oi).getListElementObjectInspector());
+    }
+
+    public static boolean isNumberListListOI(@Nonnull final ObjectInspector oi) {
+        return isListOI(oi)
+                && isNumberListOI(((ListObjectInspector) oi).getListElementObjectInspector());
```

Comment:
okey as it is. BTW, better to avoid a long single line condition for readability. This case is okey.